operations/research-and-development/product/release-process/release-scorecard-definitions.md

```diff
@@ -124,7 +110,22 @@
           AND created &lt; &quot;END&quot; AND resolution not in (Duplicate, &quot;Cannot
           Reproduce&quot;, &quot;Won&apos;t Fix&quot;) AND (fixVersion not in (latestReleasedVersion(),
           earliestUnreleasedVersion()) OR fixVersion is EMPTY)</p>
-        </td>
+      </td>
+    </tr>
+    <tr>
+      <td style="text-align:left">Total valid bugs found after RC1 is cut</td>
+      <td style="text-align:left">
+        <p>After closing current release, adjust dates as per above, and use this
+          Jira query:</p>
+        <ol>
+          <li>Check Jira timezone + Pre-release timezone and make sure times match</li>
```

Comment:
@amyblais is this an outdated reference to the old Pre-release name for the server, or referring to something else?

Reply:
Yes, it was outdated info. Updated now 👍